build_image.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+CHEF_VERSION="provisionerless"
+TEMPLATE=
+PPC64LE_PACKER=~/bento/packer/bin/packer
+X86_64_PACKER=/usr/local/bin/packer
```

Comment:
I don't like how these are hard coded. Why isn't packer installed in the same location on pp64le?